docs/site/src/demos/tables/EnhancedTable.js

```diff
@@ -31,6 +31,18 @@ function createData(name, calories, fat, carbs, protein) {
   return { id: counter, name, calories, fat, carbs, protein };
 }
 
+function createColumnData(id, numeric, padding, label) {
```

Comment:
WDYT to something like `createHeadData` since we have `createData` for the table body? `createColumnData` vs `createData` doesn't provide enough distinction.